getAllFeatureFlags/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"log"
+	"encoding/json"
+	"net/http"
+)
+
+func getAllFeatureFlags(db *dynamodb.DynamoDB) ([]utils.FeatureFlagResponse, error) {
+
+	input := &dynamodb.ScanInput{
+		TableName: aws.String(database.GetFeatureFlagTableName()),
```

Comment:
Please use the `GetTableName()` instead
https://github.com/Real-Dev-Squad/feature-flag-backend/blob/ea0e2a48cdecd926bc615308e56c03ec1c972436/database/dynamodb.go#L61-L71

Reply:
updated